fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/DimensionColumn.java

```diff
@@ -42,16 +40,6 @@ public Dimension getDimension() {
     }
 
     @Override
-    public int hashCode() {
-        return dimension.hashCode();
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        return (o instanceof DimensionColumn  && Objects.equals(((DimensionColumn) o).getDimension(), dimension));
-    }
```

Comment:
Removing `equals` and `hashCode` so that retrieving `DimensionColumn` and `MetricColumn` by `Column` base class is possible.